app/config/services.php

```diff
@@ -186,4 +186,30 @@
         'optional' => true,
         'icon' => '/images/services/graphql.png',
     ],
+    'vcs' => [
+        'key' => 'vcs',
+        'name' => 'VCS',
+        'subtitle' => 'The VCS service allows you to interact with providers like GitHub, GitLab etc.',
+        'description' => '',
+        'controller' => 'api/vcs.php',
+        'sdk' => false,
+        'docs' => false,
+        'docsUrl' => '',
+        'tests' => false,
+        'optional' => true,
+        'icon' => '',
+    ],
+    'vcs_map' => [
```

Comment:
Why do we have 2 new controllers? we should have implemented only `v1/vcs` as a single source of truth for all VCS integrations.

Reply:
I don't see another controller, so this might just be a mistake?

Reply:
Yes, I accidentally added `collections` details instead of `controllers`.